crates/provider-sqldb-postgres/README.md

```diff
@@ -0,0 +1,181 @@
+
+        # Link the sqldb-provider to the component, specifying the postgres cluster URL
+        - type: link
+          properties:
+            target: sqldb-postgres
+            namespace: wasmcloud
+            package: sqldb-postgres
+            interfaces: [migrations, managed-raw, managed-prepared]
+            target_config:
+              - name: pg
+                properties:
+                  url: postgres://127.0.0.1:5432
+```
+
+</details>
+
+[wadm]: https://github.com/wasmCloud/wadm
+
+
+## 📑 Named configuration Settings
+
+As connection details are considered sensitive information, they should be specified via named config available to the provider *at startup*, rather than via link definitions.
+
+> ![NOTE]
+> Named configuration can be specified by:
+>
+> - Using appropriate options on `wash config put`
+
+| Property                      | Example                     | Description                                                                |
+|-------------------------------|-----------------------------|----------------------------------------------------------------------------|
+| `MANAGED_URL`                 | `postgres://localhost:5432` | URL to use for all managed connections (overrides `*_HOST`, `*_PORT`, etc) |
+| `MANAGED_HOST`                | `localhost`                 | Hostname for all managed connections                                       |
+| `MANAGED_PORT`                | `5432`                      | Port for all managed connections                                           |
+| `MANAGED_USERNAME`            | `postgres`                  | Username for all managed connections                                       |
+| `MANAGED_PASSWORD`            | `postgres`                  | Password for all managed connections                                       |
+| `MANAGED_TLS_REQUIRED`        | `false`                     | Whether TLS should be required for al managed connections                  |
+| `PROFILE_<name>_URL`          | `postgres://some-db:5432`   | URL to use for all named profile connection `<name>`                       |
+| `PROFILE_<name>_HOST`         | `localhost`                 | Hostname to use for named profile connection `<name>`                      |
+| `PROFILE_<name>_PORT`         | `5432`                      | Port to use for named profile connection `<name>`                          |
+| `PROFILE_<name>_USERNAME`     | `postgres`                  | Username to use for named profile connection `<name>`                      |
+| `PROFILE_<name>_PASSWORD`     | `postgres`                  | Password to use for named profile connection `<name>`                      |
+| `PROFILE_<name>_TLS_REQUIRED` | `false`                     | Whether to require TLS for named profile connection `<name>`               |
+
+> ![NOTE]
+> As multiple components will connect to the same provider, the primary method of configuring connection "profiles" is via ENV variables
```

Comment:
Especially for future compatibility when this could be a component, this should be done through provider config and not environment variables

Reply:
However, I can't see in code where it is loading anything from the environment, so I am unsure if this is accurate or not?

Reply:
Ah sorry yeah this isn't accurate, I'm referring to the variables above but that's provider config -- they just *look* like ENV variables, will remove!